File: dataset_reduzido/avatar/Python/Code/codeforces_621_A.py
n = int ( input ( ) )
lst = list ( map ( int , input ( ).split ( ) ) )
lst.sort ( )
lst.reverse ( )
od = 0
for i in lst :
    if ( i & 1 ) :
        od += 1
sum = 0
ok = 0
for i in range ( n ) :
    if ( lst [ i ] % 2 == 0 ) :
        sum += lst [ i ]
    else :
        if ( ok or od > 1 ) :
            sum += lst [ i ]
            od -= 1
            ok = not ok
print ( sum )

File: output/LLaMa_repaired_1/avatar/Python/Java/codeforces_621_A.java
import java.util.Arrays;
import java.util.Scanner;

public class codeforces_621_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] strArray = scanner.nextLine().split(" ");
        int[] lst = new int[n];
        for (int i = 0; i < n; i++) {
            lst[i] = Integer.parseInt(strArray[i]);
        }
        Arrays.sort(lst);
        reverseArray(lst);
        int od = 0;
        for (int i : lst) {
            if ((i & 1) != 0) {
                od++;
            }
        }
        long sum = 0;
        boolean ok = false;
        for (int i = 0; i < n; i++) {
            if (lst[i] % 2 == 0) {
                sum += lst[i];
            } else {
                if (ok || od > 1) {
                    sum += lst[i];
                    od--;
                    ok = !ok;
                }
            }
        }
        System.out.println(sum);
    }

    public static void reverseArray(int[] array) {
        int left = 0;
        int right = array.length - 1;
        while (left < right) {
            int temp = array[left];
            array[left] = array[right];
            array[right] = temp;
            left++;
            right--;
        }
    }
}